charts/ccsm-helm/charts/identity/Chart.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v2
+description: Identity Helm Chart for Kubernetes
+name: identity
+version: 0.0.25
+type: application
+icon: https://raw.githubusercontent.com/camunda-cloud/camunda-cloud-documentation/master/static/img/camunda-operate-gradient.png
```

Let's use the IAM logo for now and update it if we rename it or create an own logo for Identity

```suggestion
icon: https://raw.githubusercontent.com/camunda-cloud/camunda-cloud-documentation/master/static/img/iam.png
```